### ISSUE TITLE
Buy Menu Bug

### ISSUE THREAD

**Issue by CplShephard** [Velaron/cs16-client#103]
When you press the buy menu button just as you are leaving the purchasing area, the touch controls disappear.

https://github.com/user-attachments/assets/d3afcabf-789d-4b1b-a5bb-26e9ebbb8f52

**Comment by Vladislav4KZ** [comment]
You are using the latest build of CS16Client from GitHub, not from Google Play Archive?

I encountered a similar bug, but I fixed it a long time ago.

GitHub releases are here: https://github.com/Velaron/cs16-client/releases/tag/continuous

**Comment by 21Shephard** [comment]
> Are you using the latest build of CS16Client from GitHub, not from Google Play Archive?
> 
> I encountered a similar bug, but I fixed it a long time ago.
> 
> GitHub releases are here: https://github.com/Velaron/cs16-client/releases/tag/continuous

Yes,I am currently using latest build of CS16Client and latest Xash3d FWGS build of github.Idk why this happening.
